test/k8sT/Chaos.go

```diff
@@ -47,6 +47,8 @@ var _ = Describe("K8sChaosTest", func() {
 
 		ExpectCiliumReady(kubectl)
 		ExpectKubeDNSReady(kubectl)
+		By("Waiting for all etcd-operator pods are ready")
```

Comment:
I think that this By message is not in the right place, that By message should be into `ExpectETCDOperatorReady`

Reply:
@eloycoto it's not possible to have By messages in the `assertionHelpers.go`

Reply:
Why not? could you explain why?

Reply:
I thought it was not possible but it turns out it missed imports.